cmd/soroban-rpc/internal/daemon/daemon.go

```diff
@@ -162,8 +162,18 @@ func MustNew(cfg config.LocalConfig, endpoint string, adminEndpoint string) *Dae
 	}
 	dbConn := dbsession.RegisterMetrics(session, "soroban_rpc", "db", prometheusRegistry)
 
-	eventStore := events.NewMemoryStore(cfg.NetworkPassphrase, cfg.EventLedgerRetentionWindow)
-	transactionStore := transactions.NewMemoryStore(cfg.NetworkPassphrase, cfg.TransactionLedgerRetentionWindow)
+	eventStore := events.NewMemoryStore(
+		prometheusRegistry,
+		"soroban_rpc",
```

Comment:
I think there is no added value in passing "soroban_rpc" all around the place.
Instead, in /internal/metrics, create a file named `metrics.go` that would have a single const
```golang
const (
const PrometheusNamespacesSorobanRPC = "soroban_rpc"
)
```
and just use it everywhere instead of passing it on the stack.